landoapi/api/transplants.py

```diff
@@ -24,25 +23,22 @@
     get_release_managers,
     get_sec_approval_project_phid,
     get_secure_project_phid,
-    get_testing_tag_project_phids,
     get_testing_policy_phid,
+    get_testing_tag_project_phids,
     project_search,
 )
-from landoapi.repos import (
-    Repo,
-    get_repos_for_env,
-)
+from landoapi.repos import Repo, get_repos_for_env
```

Comment:
Changes like this are generally bad IMO - it's better to have each import on it's own line so the blame history is preserved in VCS.

Reply:
Are you referring to blame history in general, or the blame history as it relates to this PR?

I agree that it is nicer to have each import on its own line, we currently mix the two anyway, but I am not opposed to changing the settings here.